### PR TITLE
Adding CI feature with .travis.yml file

### PR DESCRIPTION
**Pull request by carmat88**
File .travis.yml is being created in order to add the CI feature to the docs repository and testing correctness of documentation

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: python # (Ansible)
+
+sudo: required
```
Comment (mcapuccini) [inline .travis.yml]: Try to see if you can avoid sudo, so it will run faster

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+install:
+
+  # Install Sphinx
+  - sudo pip install --upgrade pip
+  - sudo pip install sphinx==1.5.3
+  - sudo pip install sphinx-autobuild==0.6.0
```
Comment (mcapuccini) [inline .travis.yml]: plase try to remove this

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - sudo pip install sphinx_rtd_theme==0.2.4
```
Comment (mcapuccini) [inline .travis.yml]: please try to review this

**Review by mcapuccini — CHANGES_REQUESTED** [review]
Some minor things need to be fixed/investigated

File: .travis.yml (hunk 1 above)
Comment (carmat88) [reply .travis.yml]: Indeed, it seems that by removing sudo the test is faster

File: .travis.yml (hunk 2 above)
Comment (carmat88) [reply .travis.yml]: same here, sudo removed from the pip commands

File: .travis.yml (hunk 3 above)
Comment (carmat88) [reply .travis.yml]: same here, sudo removed

**Comment by carmat88** [comment]
CI feature is correctly working

**Pull request merged**